### PR TITLE
Fixed - Navigation issue of review from product listing when click on review count

### PR DESCRIPTION
**Pull request by niravkrish**
### Description (*)
From product listing click on review count it navigate to add review form.

### Fixed Issues (if relevant)

1. #21558: Navigation issue of review from product listing when click on review count

### Manual testing scenarios (*)

1. Manual Testing

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @niravkrish. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by slavvka** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @slavvka. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @slavvka, here is your [new Magento instance](https://pr-21559.instances.magento-community.engineering).
Admin access: [https://pr-21559.instances.magento-community.engineering/admin](https://pr-21559.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by rogyar** [comment]
Hi @niravkrish. Please, consider fixing failing tests.
Thank you

**Comment by niravkrish** [comment]
Hi @rogyar,
I have fixed all failing test. 
Please process this Pull Request.
Thank You

File: app/code/Magento/Review/view/frontend/web/js/process-reviews.js [hunk 1]
```diff
@@ -27,6 +29,14 @@ define([
...
+
+            if (hashAction) {
+                setTimeout(function () {
+                    $('html, body').animate({
+                        scrollTop: $('#' + hashAction).offset().top - 50
+                    }, 300);
+                }, 100);
+            }
         }).complete(function () {
             if (fromPages == true) { //eslint-disable-line eqeqeq
```
Comment (rogyar) [inline app/code/Magento/Review/view/frontend/web/js/process-reviews.js:41]: Hi @niravkrish. I don't think we need to provide such workaround here. 
Take a look at the `if (fromPages == true)` condition. The problem is `fromPages` variable is always undefined. As result, the condition's body that scrolls the page to the correct position (reviews) is never being executed. So, we need to figure out why `fromPages` is always undefined and provide a fix.

**Review by rogyar — CHANGES_REQUESTED** [review]

**Comment by sidolov** [comment]
@niravkrish , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

File: app/code/Magento/Review/view/frontend/web/js/process-reviews.js (hunk 1 above)
Comment (niravkrish) [reply app/code/Magento/Review/view/frontend/web/js/process-reviews.js:41]: fromPages is already there before I code.
so what should i need to do for that.

**Pull request closed without merging**